localstack/services/awslambda/lambda_api.py

```diff
@@ -170,6 +170,12 @@ def func_arn(function_name):
     return aws_stack.lambda_function_arn(function_name)
 
 
+def func_qualifier(function_name, qualifier=None):
+    arn = aws_stack.lambda_function_arn(function_name)
+    if ARN_TO_LAMBDA.get(arn).qualifier_exists(qualifier):
+        return '{}:{}'.format(arn, qualifier)
```

Comment:
I think we should `return arn` as a fallback at the end of this function (otherwise the `['Resource']` entry below could become `None`).

Reply:
Got it.